rules/windows/builtin/win_susp_raccess_sensitive_fext.yml

```diff
@@ -21,7 +21,6 @@ detection:
             - '*.bak'
             - '*.dmp'
             - '*.kirbi'
-            - '*\ntds.dit'
```

Comment:
deleted due to creating of the [win_transferring_files_with_credential_data_via_network_shares.yml](https://github.com/Neo23x0/sigma/blob/1f7b3bc9a2d4c1f743da51f4283547d741b3c0e7/rules/windows/builtin/win_transferring_files_with_credential_data_via_network_shares.yml). this stuff is now there:

```
title: Transferring files with credential data via network shares
description: Transferring files with well-known filenames (sensitive files with credential data) using network shares
author: Teymur Kheirkhabarov, oscd.community
date: 2019/10/22
references:
    - https://www.slideshare.net/heirhabarov/hunting-for-credentials-dumping-in-windows-environment
tags:
    - attack.credential_access
    - attack.t1003
logsource:
    product: windows
    service: security
detection:
    selection:
        EventID: 5145
        RelativeTargetName|contains:
            - '\mimidrv'
            - '\lsass'
            - '\windows\minidump\'
            - '\hiberfil'
            - '\sqldmpr'
            - '\sam'
            - '\ntds.dit'
            - '\security'
    condition: selection
falsepositives:
    - Transferring sensitive files for legitimate administration work by legitimate administrator
level: medium
status: experimental
```